main.js

```diff
@@ -0,0 +1,366 @@
+
+function startGame(event) {
+  cell1.innerHTML = "";
+  cell2.innerHTML = "";
+  cell3.innerHTML = "";
+  cell4.innerHTML = "";
+  cell5.innerHTML = "";
+  cell6.innerHTML = "";
+  cell7.innerHTML = "";
+  cell8.innerHTML = "";
+  cell9.innerHTML = "";
+
+  cell1.classList = "square";
+  cell2.classList = "square";
+  cell3.classList = "square";
+  cell4.classList = "square";
+  cell5.classList = "square";
+  cell6.classList = "square";
+  cell7.classList = "square";
+  cell8.classList = "square";
+  cell9.classList = "square";
+
+  xImage =
+    "<img class='character-image fade-in' src='images/x/" +
+    randomInt(7, 1) +
+    ".gif' height=90% width=90% />";
+  oImage =
+    "<img class='character-image fade-in' src='images/o/" +
+    randomInt(13, 1) +
+    ".gif' height=90% width=90% />";
+
+  if (singlePlayer.checked === true) {
+    twoPlayer.disabled = true;
+    playerTwo = "Computer";
+  } else {
+    singlePlayer.disabled = true;
+    playerTwo = "Player Two";
+  }
+
+  xSquares = [];
+  oSquares = [];
+  emptySquares = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+  event.target.disabled = true;
+  gameOver = false;
+  totalSeconds = 0;
+  countTimer();
+  turnNumber = 1;
+  statusBar.textContent = "Player One's Turn";
+}
+
+/////////////Logic for making a move/////////////////
+
+function drawCell(event) {
+  if (gameOver === false) {
+    if (
+      event.target.innerHTML === "" &&
+      event.target.className != "character-image fade-in"
+    ) {
+      if (twoPlayer.checked === true || whosTurn(turnNumber) === "x") {
+        if (whosTurn(turnNumber) === "o") {
+          event.target.innerHTML = oImage;
+          oSquares.push(event.target.squareNumber);
+          removeSquare(event.target.squareNumber, emptySquares);
+        }
+        if (whosTurn(turnNumber) === "x") {
+          event.target.innerHTML = xImage;
+          xSquares.push(event.target.squareNumber);
+          removeSquare(event.target.squareNumber, emptySquares);
+        }
+        anybodyWonYet(xSquares, oSquares);
+
+        turnNumber += 1;
+
+        whosTurn(turnNumber);
+
+        if (gameOver === false && singlePlayer.checked) {
+          setTimeout(cpuMove, 750);
+        } 
+      }
+    } else {
+      statusBar.textContent =
+        "That cell is full, but there's plenty to go around!";
+    }
+  }
+}
+
+/////////////Logic for CPU move////////////////
+
+function cpuMove() {
+  let randomSquare = Math.floor(Math.random() * emptySquares.length);
+  cpuSquare = document.getElementById("cell-" + emptySquares[randomSquare]);
+  cpuSquare.innerHTML = oImage;
+  oSquares.push(cpuSquare.squareNumber);
+  removeSquare(cpuSquare.squareNumber, emptySquares);
+  anybodyWonYet(xSquares, oSquares);
+  turnNumber += 1;
+  whosTurn(turnNumber);
+}
+
+////////////Checks if anybody has three in a row, and ends the game, this could definately be broken up into several functions, I'm sorry!!!//////////////
+
+function anybodyWonYet(xSquares, oSquares) {
```

Comment:
This function is fine, checking if x has won and then checking if o has won, it could just use a few more comments for readability